src/combinations.rs

```diff
@@ -47,9 +47,7 @@ impl<I: Iterator> Combinations<I> {
     pub fn k(&self) -> usize { self.indices.len() }
 
     /// Returns the (current) length of the pool from which combination elements are
-    /// selected. This value can change between invocations of [`next`].
-    ///
-    /// [`next`]: #method.next
+    /// selected. This value can change between invocations of [`next`](Combinations::next).
```

Comment:
Links to `Combinations::next` (not `Iterator::next`). Is this canonical?

Reply:
I'm not sure, my first thought was that it should point to the `itertools` documentation instead of sending to the general trait method.